apps/server/src/modules/learnroom/controller/api-test/course.api.spec.ts

```diff
@@ -114,4 +118,24 @@ describe('Course Controller (API)', () => {
 			expect(response.header['content-disposition']).toBe('attachment;');
 		});
 	});
+
+	describe('[POST] /courses/import', () => {
+		const setup = async () => {
+			const teacher = createTeacher();
+			const course = await readFile('./apps/server/test/assets/common-cartridge/us_history_since_1877.imscc');
+
+			return { teacher, course };
+		};
+
+		it('should import course', async () => {
+			const { teacher, course } = await setup();
+			await em.persistAndFlush([teacher.account, teacher.user]);
+			em.clear();
+
+			const loggedInClient = await testApiClient.login(teacher.account);
+			const response = await loggedInClient.postWithAttachment('import', 'file', course, 'us_history_since_1877.imscc');
```

Comment:
imscc filename should be stored in a constant, easier to work with in case you have add  another test latter

Reply:
Done.